routes/data.js

```diff
@@ -34,6 +34,52 @@ async function getAcceptedGategoriesByRef(ref, fieldName) {
   return results;
 }
 
+/**
+ * Fetches all organizations from the database.
+ * @return {array} The list of all organizations.
+ */
+exports.getAllOrganizations = async function () {
+  const organizations = await firestore.collection(resolveCollectionName('Organizations')).get();
+
+  const parsedOrganizations = [];
+
+  organizations.docs.forEach((doc) => {
+    const data = doc.data();
+    data['id'] = doc.id;
+
+    parsedOrganizations.push(data);
+  });
+
+  return parsedOrganizations;
+};
+
+/**
+ * Queries the database for organizations with the given item category.
+ * @param {String} filter The category name to filter organizations by.
+ * @return {array} The list of filtered organizations.
+ */
+exports.getFilteredOrganizations = async function (filter) {
+  filter = filter.toLowerCase();
+
+  const categoryReference = await firestore
+    .collection(resolveCollectionName('Categories'))
+    .doc(filter);
+  const acceptedCategories = await getAcceptedCategoriesByRef(categoryReference, 'category');
+
+  const organizations = [];
+
+  for (const key in acceptedCategories) {
+    if (Object.prototype.hasOwnProperty.call(acceptedCategories, key)) {
```

Comment:
This line would be redundant. `acceptedCategories` will always have `key` because the for-in loop only sets `key` to a value that exists in `acceptedCategories`

Reply:
When you use a `for in` loop with the linter, it suggests that you wrap the for loop in this if statement.
https://stackoverflow.com/questions/1963102/what-does-the-jslint-error-body-of-a-for-in-should-be-wrapped-in-an-if-statemen